.github/workflows/workflow.yml

```diff
@@ -471,34 +438,25 @@ jobs:
     - name: Checkout code
       if: github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags')
       uses: actions/checkout@v2
-    - name: Configure gcloud
+    - name: Install linkerd CLI
```

Comment:
So it looks like `gcloud .. create/delete` only need the Linkerd CLI to be installed for the action's `getClusterName` which relies on `linkerd version --client --short`.

In `cloud_integration` this makes sense, but it's not stricly necessary to have the CLI installed for `cloud_cleanup`. Maybe instead of the descriptions requiring that a CLI is installed, they just pass in `$TAG` as an input? That way we don't need to install the CLI since we can already rely on `$TAG` having the correct value.

Reply:
Yeah agree, I think decoupling the gcloud github action from linkerd cli makes sense. It's good to validate the Linkerd CLI version matches repo version, but that could/should be done elsewhere. Taking `$TAG` as input a step further, maybe consider just having a `name` input to specify the cluster name.